client/trino-cli/src/main/java/io/trino/cli/Completion.java

```diff
@@ -13,42 +13,42 @@
  */
 package io.trino.cli;
 
-import com.google.common.collect.ImmutableSet;
 import org.jline.reader.Completer;
+import org.jline.reader.impl.completer.AggregateCompleter;
+import org.jline.reader.impl.completer.ArgumentCompleter;
+import org.jline.reader.impl.completer.NullCompleter;
 import org.jline.reader.impl.completer.StringsCompleter;
 
-import java.util.Set;
+import java.util.List;
 
-import static java.util.Locale.ENGLISH;
-import static java.util.stream.Collectors.toSet;
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
 
 public final class Completion
 {
-    private static final Set<String> COMMANDS = ImmutableSet.of(
-            "SELECT",
-            "SHOW CATALOGS",
-            "SHOW COLUMNS",
-            "SHOW FUNCTIONS",
-            "SHOW SCHEMAS",
-            "SHOW SESSION",
-            "SHOW TABLES",
-            "CREATE TABLE",
-            "DROP TABLE",
-            "EXPLAIN",
-            "DESCRIBE",
-            "USE",
-            "HELP",
-            "QUIT");
-
     private Completion() {}
 
     public static Completer commandCompleter()
     {
-        return new StringsCompleter(ImmutableSet.<String>builder()
-                .addAll(COMMANDS)
-                .addAll(COMMANDS.stream()
-                        .map(value -> value.toLowerCase(ENGLISH))
-                        .collect(toSet()))
-                .build());
+        return new AggregateCompleter(buildArgumentCompleter("CREATE", singletonList("TABLE")),
```

Comment:
Can we have also SCHEMA? Can we also have other statements too like `ALTER TABLE/SCHEMA`

Reply:
I initially kept the list of commands as it was before, but sure. I added those you suggested.